packages/react-magma-dom/src/components/Pagination/PageButton.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Button, ButtonColor, ButtonProps, ButtonVariant } from '../Button';
+import { BuildBorder, hoverBorder } from './Pagination';
+import { darken } from 'polished';
+import styled from '../../theme/styled';
+import { ThemeContext } from '../../theme/ThemeContext';
+
+export interface PaginationProps extends ButtonProps {
+  isSelected?: boolean;
+}
+
+function typeSize(props) {
+  if (props.size === 'large') {
+    return `${props.theme.typeScale.size05.fontSize}`;
+  }
+  return `${props.theme.typeScale.size03.fontSize}`;
+}
+
+function buttonSize(props) {
+  switch (props.size) {
+    case 'large':
+      return `${props.theme.spaceScale.spacing11}`;
+    default:
+      return `${props.theme.spaceScale.spacing09}`;
+  }
+}
+
+function boxShadowColor(props) {
+  if (props.color === 'primary') {
+    if (props.isInverse) {
+      return `-${props.theme.spaceScale.spacing01} 0 0 ${props.theme.colors.neutral08}`;
+    }
+    return `-${props.theme.spaceScale.spacing01} 0 0 ${props.theme.colors.primary}`;
+  }
+}
+
+function hoverBoxShadowColor(props) {
+  if (props.color === 'primary') {
+    if (props.isInverse) {
+      return `-${props.theme.spaceScale.spacing01} 0 0 ${props.theme.colors.neutral08}`;
+    }
+    return `-${props.theme.spaceScale.spacing01} 0 0 ${darken(
+      0.1,
+      props.theme.colors.primary
+    )}`;
+  }
+}
+
+const StyledPageButton = styled(Button)`
+  border: none;
+  border-top: ${BuildBorder};
+  border-right: ${BuildBorder};
+  border-bottom: ${BuildBorder};
```

Comment:
Instead of setting border to none and then calling a `BuildBorder` function on each of the sides, could you get the same effect by doing `border-width: 2px 2px 2px 0;` ?  I think if we do that, we wouldn't need the `BuildBorder` function at all.